src/data/mv_polynomial/basic.lean

```diff
@@ -125,6 +125,10 @@ instance [monoid R] [comm_semiring S₁] [distrib_mul_action R S₁] [distrib_mu
 add_monoid_algebra.is_central_scalar
 instance [comm_semiring R] [comm_semiring S₁] [algebra R S₁] : algebra R (mv_polynomial σ S₁) :=
 add_monoid_algebra.algebra
+-- Register with high priority to avoid timeout in `data.mv_polynomial.pderiv`
+instance is_scalar_tower' [comm_semiring R] [comm_semiring S₁] [algebra R S₁] :
+  is_scalar_tower R (mv_polynomial σ S₁) (mv_polynomial σ S₁) :=
+is_scalar_tower.right
```

Comment:
Why do we need this instance? Do we understand the bug?

Reply:
As far as I understand, Lean tries to use https://leanprover-community.github.io/mathlib_docs/data/mv_polynomial/basic.html#mv_polynomial.is_scalar_tower and fails.